Audacia.Log.AspNetCore/README.md

```diff
@@ -67,7 +67,55 @@ public IConfiguration Configuration { get; set; }
 public void ConfigureServices(IServiceCollection services)
 {
 	services.ConfigureActionContentLogging(Configuration);
-	services.AddControllers(x => x.Filters.Add<LogActionFilterAttribute>());
+    services.AddClaimsTelemetry();
+	services.AddControllers(x => x.Filters.Add<LogClaimsActionFilterAttribute>());
+}
+```
+
+#### AddClaimsTelemetry()
+This extension method is overloaded and if you want to pass in your own implementation of "CustomAdditionalClaimsTelemetryProvider" to select which properties you would like to pass on, do it like below:
+
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+	services.ConfigureActionContentLogging(Configuration);
+    services.AddClaimsTelemetry(new CustomAdditionalClaimsTelemetryProvider((user) => 
+    {
+          return new List<(string Name, string Data)>
+          {
+              ("customproperty", user.Claims.Where(c => c.Type == "customproperty").Single().Value)
+          };
+    }))
+	services.AddControllers(x => x.Filters.Add<LogClaimsActionFilterAttribute>());

```

Comment:
I can't see why we would want to log out different claims on a per-endpoint basis, which is assumedly why this attribute exists, because your token (& therefore claims) will be the same for all of them.

I'd be in favour of removing this attribute in which case, but happy to discuss